infra/aws/terraform/prow-build-cluster/docs/Debug.md

```diff
@@ -0,0 +1,30 @@
+## SSH Access EKS Nodes
+
+To access cluster nodes via ssh, first you need to provision bastion host:
+
+```bash
+TF_ARGS="-var bastion_install=true" make apply
```

Comment:
Let's recommend doing this via `.tfvars` so that we don't lose bastion if someone forgets the flag.

Reply:
I made it this way on purpose to point out the ephemeral nature of bastion host. In case someone removes it, it's easy to provision it again.
Another reason is that we don't have any gitops in place for applying terraform, so at the end of the say someone has to run the script manually.
But I'm also fine having this placed in `.tfvars`.